config/software/perl.rb

```diff
@@ -22,6 +22,8 @@
 
 default_version "5.18.1"
 
+dependency "gecode"
```

Comment:
Why is `gecode` needed? This is a very expensive (it has a long and CPU-intensive compilation) library to compile.

Reply:
I originally performed the Perl compilation testing within an EL 8 machine.  There were two commits that I originally left in this branch that were related to getting Perl to compile properly on EL 8; one of those was specifying the `gecode` dependency.

I've since removed those two commits and did a minor clean up of some sed lines in my Perl patch files that @jeremiahsnapp pointed out.

This build should (hopefully) show everything is still working without the EL 8 commits:

https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/407

Reply:
nice catch @schisamo totally missed that myself!

Reply:
yep, build 407 seems to show that it is fine without the EL 8 extras, so I *think* this PR is ready to be merged